examples/llm/benchmarking.md

```diff
@@ -0,0 +1,333 @@
+<!--
+SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+SPDX-License-Identifier: Apache-2.0
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# LLM Deployment Benchmarking Guide
+
+This guide provides detailed steps on benchmarking Large Language Models (LLMs) in single and multi nodes configurations.
+
+> [!NOTE]
+> We advice trying out the [LLM Deployment Examples](./README.md) before benchmarking.
+
+## Build the Benchmarking Image
+
+1\. Clone Dynamo repository
+```bash
+git clone --single-branch --depth=1 -b main https://github.com/ai-dynamo/dynamo.git
+```
+
+2\. Build vLLM image
+```bash
+./container/build.sh
+```
+Note: Make sure to run it in the `dynamo` directory cloned.
+
+## Disaggregated Single Node Benchmarking
+
+One H100 80GB x8 node is required for this setup.
+
+With the Dynamo repository and the benchmarking image available, perform the following steps:
+
+1\. Run benchmarking container
+```bash
+./container/run.sh -it
+```
+Note: Make sure to run it in the `dynamo` directory cloned.
+
+2\. Download model
+```bash
+huggingface-cli download neuralmagic/DeepSeek-R1-Distill-Llama-70B-FP8-dynamic
```

Comment:
do we want to download outside the container for reuse and mount the directory during run? - or can use mount workspace on run in order to get outputs / save model?

Reply:
Sure, updated guide to advice users to download the model outside the container, and mount the huggingface/hub during run.